.github/workflows/cmake-builds.yml

```diff
@@ -32,20 +32,20 @@ jobs:
         cd cmake/build-ninja
         cpack -G DEB -C Release
     - name: Upload DEB
-      uses: actions/upload-artifact@v3
+      uses: actions/upload-artifact@v4
       with:
         name: simh-4.1.0-x86_64-${{matrix.os}}.deb
         path: cmake/build-ninja/simh-4.1.0-x86_64-${{matrix.os}}.deb
 
 
   cmake-macOS:
-    name: macOS
+    name: macOS 12+
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [macos-12, macos-11]
+        os: [macos-12, macos-13, macos-14]

```

Comment:
In the other file you have `macos-latest`, would that make sense here?

Also, what does it mean to list several OS versions?  Does it run on all of them, or just on whichever one (if any) is available?

Reply:
See previous comment.

Reply:
Artifacts, artifacts, artifacts!!!

Reply:
BTW: The image name, e.g. "macos-14", appears in the artifact name. So, "-latest" might be awkward.

Reply:
I understand the difference now, that makes sense.  A comment might help.

PS. "Artefact" is the original spelling, still used in Britain and Australia and also how the corresponding word is spelled in several other languages.  :-)

Reply:
Duly noted.